docusaurus/src/components/CopyPageButton/CopyPageButton.jsx

```diff
@@ -0,0 +1,124 @@
+    navigator.clipboard.writeText(pageContent);
+    setIsCopied(true);
+    setTimeout(() => {
+      setIsCopied(false);
+    }, 2500);
+  };
+
+  return (
+    <div className={styles.container}>
+      <Button className={styles.copyButton} onClick={handleCopyPage}>
+        {isCopied ? (
+          <>
+  
+            <FontAwesomeIcon
+              icon={faCheck}
+              className={styles.menuItemIcon}
+            />{" "}
+            <span className={styles.menuItemTitle}>
+              Copied
+            </span>
+          </>
+        ) : (
+          <>
+            <FontAwesomeIcon icon={faClone} className={styles.menuItemIcon} />{" "}
+            <span className={styles.menuItemTitle}>
+              {" "}
+              Copy Page
+            </span>
+          </>
+        )}
+      </Button>
+      <Menu as="div">
+        <MenuButton className={styles.dropdownButton}>
+          <FontAwesomeIcon icon={faChevronDown} />
+        </MenuButton>
+        <MenuItems className={styles.menuItems}>
+          <MenuItem>
+            <Button className={styles.menuItem} onClick={handleCopyPage}>
+              <FontAwesomeIcon icon={faClone} className={styles.menuItemIcon} />
+              <div className={styles.menuItemContent}>
+                <div className={styles.menuItemTitle}>Copy page</div>
+                <div className={styles.menuItemDescription}>
+                  Copy page as Markdown for LLMs
+                </div>
+              </div>
+            </Button>
+          </MenuItem>
+          <MenuItem>
+            <Link className={styles.menuItem} to={mdPath} target="_blank">
+              <FontAwesomeIcon
+                icon={faMarkdown}
+                className={styles.menuItemIcon}
+              />
+              <div className={styles.menuItemContent}>
+                <div className={styles.menuItemTitle}>View as Markdown</div>
+                <div className={styles.menuItemDescription}>
+                  View this page as plain text{" "}
+                  <FontAwesomeIcon
+                    icon={faArrowUpRightFromSquare}
+                    className={styles.inlineIcon}
+                  />
+                </div>
+              </div>
+            </Link>
+          </MenuItem>
+          <MenuItem>
+            <Link
+              className={styles.menuItem}
+              href={`https://claude.ai/new?q=Read+from+${mdPath}+so+I+can+ask+questions+about+it`}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <FontAwesomeIcon icon={faRobot} className={styles.menuItemIcon} />
+              <div className={styles.menuItemContent}>
+                <div className={styles.menuItemTitle}>Open in Claude</div>
+                <div className={styles.menuItemDescription}>
+                  Ask questions about this page
+                </div>
+              </div>
+            </Link>
+          </MenuItem>
```

Comment:
@ian-at-airbyte  let me know what you think about this option. I saw it here: https://trigger.dev/docs/tasks/schemaTask and thought it could be useful, but happy to remove it/change it